### DISCUSSION TITLE
I couldn't find spline interpolation in groops. How can I perform curve interpolation?

### DISCUSSION THREAD

**Discussion by zdwangjian5** [groops-devs/groops#291] [Q&A]
I couldn't find spline interpolation in groops. How can I perform curve interpolation? I can import the eigen library. But the coding style is quite different.  Could you please let me know how you handle this? Thank you.

**Comment by tmayerguerr** [comment]
Hi,
Sorry, your question is unfortunately very unspecifically formulated. Do you want to implement a new method in instrumentResample? In this case, you must write a new class/file in `source/classes/interpolatorTimeSeries` and include it in `source/classes/interpolatorTimeSeries/interpolatorTimeSeries.cpp`.
regards
Torsten

**Reply by zdwangjian5** [reply]
Thank you for your reply! I just want to know how spline interpolation is done in groops and if there is an existing implementation available.
